packages/glob/src/glob.ts

```diff
@@ -26,12 +26,13 @@ export async function create(
  */
 export async function hashFiles(
   patterns: string,
-  options?: HashFileOptions
+  options?: HashFileOptions,
+  verboseMode: Boolean = false
```

Comment:
```suggestion
  verbose: Boolean = false
```
nit: can we just go with verbose as the arg, may need to push up other changes to make this work